src/cashflow.ts

```diff
@@ -9,32 +11,39 @@ const defaultRefillFactor = 20;
 const defaultRefillThresholdFactor = 8;
 
 /** The amount of tokens that will be sent to the user */
-export function creditAmount(token: TokenTicker): number {
+export function creditAmount(token: TokenTicker, factor: number = 1): Amount {
+  // Load this from connection?
+  const fractionalDigits = Number.parseInt(process.env.FAUCET_FRACTIONAL_DIGITS || "0", 10);
```

Comment:
I think we should hardcode this to one value per codec (codec.ts).

```ts
export function codecDefaultFractionalDigits(codec: Codec): number {
  switch (codec) {
    case Codec.Bns:
      return 9; // fixed for all weave tokens
    case Codec.Lisk:
      return 8; // only LSK supported
    default:
      throw new Error("Unknown codec");
  }
}
```

Reply:
:+1: